docs/guides/qiskit-addons.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Qiskit addons
+description: Understand the Qiskit addon tools , which help you build utility-grade quantum workflows.
+---
+
+# Qiskit addons
+
+Qiskit addons are a collection of modular tools for building utility-scale workloads powered by Qiskit. These tools are derived from advanced research capabilities and are designed to be stackable with other tools as part of a [Qiskit pattern.](./intro-to-patterns)
+
+## Mapping tools
+
+### Multiproduct formulas
+
+Multiproduct formulas (MPF) reduce the Trotter error of Hamiltonian dynamics through a weighted combination of several circuit executions.
```

Comment:
There will be a hyphen added to the name of this addon.

CC @mrossinek, @jenglick 
```suggestion
### Multi-product formulas

Multi-product formulas (MPF) reduce the Trotter error of Hamiltonian dynamics through a weighted combination of several circuit executions.
```